eclair-core/src/test/resources/integration/bitcoin.conf

```diff
@@ -1,11 +1,13 @@
 regtest=1
+noprinttoconsole=1 
 server=1
 port=28333
-rpcport=28332
+regtest.rpcport=28332
```

Comment:
looks weird to only have this field prefixed, can't we instead make a clean `regtest` section?